Yafc.Parser/Data/FactorioDataDeserializer.cs

```diff
@@ -333,15 +333,16 @@ private static float ParseEnergy(string? energy) {
             float energyBase = float.Parse(energy[..^2]);
 
             switch (energyMul) {
-                case 'k':
-                case 'K': return energyBase * 1e-3f;
+                case 'k': return energyBase * 1e-3f;
```

Comment:
You removed the `K` case because it's no longer used?

Reply:
no longer supported

changelog excerpt:
> Added 'R' (ronna) and 'Q' (quetta) SI prefixes.
> Removed 'K' from allowed SI prefixes - use 'k' instead.